recipes/sqlcipher/all/conanfile.py

```diff
@@ -123,7 +123,11 @@ def _build_visual(self):
             nmake_flags.append("USE_CRT_DLL=1")
         if self.settings.build_type == "Debug":
             nmake_flags.append("DEBUG=2")
-        nmake_flags.append("FOR_WIN10=1")
+        # It is very unlikely that __stdcall convetions are really wanted for 32bit windows
+        # It would also not work with tcl extensions are enabled, so deactivate it
+        # Ff someone really needs that, please feel free to add it as on option for 32bit msvc builds only
```

Comment:
```suggestion
        # It is very unlikely that the __stdcall convention is really wanted for 32bit windows
        # It would also not work when tcl extensions are enabled, so deactivate it
        # If someone really needs that, please feel free to add it as on option for x86 only
```

Reply:
Thanks for improving !